docker/Dockerfile

```diff
@@ -1,19 +1,16 @@
-# Creates a Docker image that is ready to execute tes compliance suite
+# Creates a Docker image that is ready to execute tes compliance suite                                                                                     Dockerfile                                                                                                 # Creates a Docker image that is ready to execute tes compliance suite
+
+FROM python:3.8-slim AS build-env
 
-FROM ubuntu:latest AS build-env
 WORKDIR /app
 
 # Install TES Compliance suite dependencies
 # Git is needed for the clone and jq is helpful with parsing results if needed
-RUN apt update
 ENV DEBIAN_FRONTEND=noninteractive
-RUN apt install -y software-properties-common
-
-RUN add-apt-repository ppa:deadsnakes/ppa
 RUN apt update
-RUN apt -y install python3.8 python3.8-distutils python3-pip git jq
+RUN apt -y install git jq
 
-RUN git clone https://github.com/elixir-cloud-aai/tes-compliance-suite.git 
+RUN git clone https://github.com/elixir-cloud-aai/tes-compliance-suite.git
```

Comment:
This will of course mean that every image may be different, not just with respect to the Python and OS versions, but also with respect to the actual test suite. We could of course check out a specific commit here, but perhaps it's best to leave it for now. However, @LakiG: do you want to create issues for publishing the tool and creating (semantically) versioned releases? Either on GitHub, PyPI (for easier installation) or both.

Reply:
alternatively the suite could be published as a python library that we could install using pip? (just an idea)

Reply:
Yup, that's what I meant (pip installs from PyPI). @LakiG: You can have a look at the [FOCA](https://github.com/elixir-cloud-aai/foca) package to check for some templating on how to write the `setup.py` file. It's also a good reference for versioning (you need to make sure that there's a single source of truth for a version).

Reply:
Sure. Publishing versioned suites for PyPI and GitHub will be better. We can work on that once we have separated the test runner and tests. The suite will commence its primary function from v1.0.0 then.

Reply:
It's really no big deal publishing it, so maybe consider doing it sooner. Sometimes you think release 1.0.0 is near and then a year later you still don't quite seem to be there. In fact, 1.0.0 is the point where an API is considered stable. And I'm not sure we are quite there yet. Or rather, I'm pretty sure that we are not ;-)

Nothing wrong with having a 0.1.0 or whatever prerelease on PyPI. Some very successful projects stay in that state forever :)